graylog2-web-interface/src/pages/LUTCachesPage.jsx

```diff
@@ -22,16 +23,34 @@ const LUTCachesPage = React.createClass({
 
   mixins: [
     Reflux.connect(LookupTableCachesStore),
+    Reflux.connect(LookupTablesStore, 'tableStore'),
   ],
 
   componentDidMount() {
     this._loadData(this.props);
+
+    this.errorStatesTimer = setInterval(() => {
```

Comment:
Shouldn't we avoid querying the server for error states until we can provide it? At least from the PR description I think we are still not ready for it and we can save some API calls until then.